### DISCUSSION TITLE
Multi user implemented?

### DISCUSSION THREAD

**Discussion by jfhernandeze** [anse-app/anse#53] [Ideas]
Hi, if I want to implement a multi user, like provinding this for my company and each employee to have their user and all use the same pi key but to have differetn historic conversations , can I do this?

**Comment by CNSeniorious000** [comment]
Sure, historic conversations are saved in the browser's IndexedDB locally, with no relation to the API key.

**Comment by ddiu8081** [comment]
We have no plans to provide user management features for now. You can fork and modify it freely!
